assets/blocks/restricted-content/index.js

```diff
@@ -0,0 +1,115 @@
+import { __ } from '@wordpress/i18n';
+import { Icon } from '@wordpress/components';
+import edit from './edit';
+import save from './save';
+import unenrolledMetadata from './unenrolled/block';
+import enrolledMetadata from './enrolled/block';
+import { createBlock } from '@wordpress/blocks';
+
+/**
+ * Helper method to create transform objects.
+ *
+ * @param {string} blockName        The block name to transform to.
+ * @param {string} inverseBlockName The name of the block which is the inverse of the one to transform to.
+ */
+const createWrapTransform = ( blockName, inverseBlockName ) => ( {
+	from: [
+		{
+			type: 'block',
+			isMultiBlock: true,
+			blocks: [ '*' ],
+			__experimentalConvert: ( blocks ) => {
```

Comment:
There is a `transform(attributes, innerBlocks)` function as a stable API, any reason this one is used?

Reply:
Yes transform includes the attributes and inner blocks only but we need the plugin name in convert. It also supports better multiple selections. It also seems to be the plan to [deprecate transform in favor of convert](https://github.com/WordPress/gutenberg/pull/19401) so I thought that it is safe enough to use.